pkg/kafka/policy_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2017 Authors of Cilium
```

Comment:
It would be nice to have a separate unit test for each helper function in policy.go, but this can be done later.